File: BackEnd/postman/Consultorio.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Consultorio'
data={
  'accion':'crear',
  'idPiso': None,
}
print postear(data,url)
print

url='Consultorio'
data={
  'accion':'listarPorPiso',
  'idPiso': None,
}
print postear(data,url)
print

url='Consultorio'
data={
  'accion':'listarPorCentro',
  'idCentroMedico': None,
  'nombreCentro': None,
}
print postear(data,url)
print

url='Consultorio'
data={
  'accion':'listarPorMedico',
  'idMedico': None,
}
print postear(data,url)
print

url='Consultorio'
data={
  'accion':'vincularMedico',
}
print postear(data,url)
print

File: BackEnd/ajax/Consultorio.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Consultorio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'idPiso': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Consultorio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTurnos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Consultorio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'vincularMedico',
    'idTurno': null,
    'idMedico': null,
    'idConsultorio': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Consultorio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPiso',
    'idPiso': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Consultorio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorMedico',
    'idMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Consultorio.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentro',
    'idCentroMedico': null,
    'nombreCentro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


